backend-api/src/functions/asyncActiveSession/tokenService/tests/tokenService.test.ts

```diff
@@ -12,6 +12,7 @@ import { mockClient } from "aws-sdk-client-mock";
 import {
```

Comment:
Do we need a test block for when `tokenVerifier.verify` returns an error result?